Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Set procps name.

diff -Naurp open-vm-tools.orig/configure open-vm-tools/configure
--- open-vm-tools.orig/configure
+++ open-vm-tools/configure
@@ -19350,7 +19350,7 @@ fi
 
 if test "$with_procps" = "yes"; then
    if test -z "$CUSTOM_PROCPS_NAME"; then
-      CUSTOM_PROCPS_NAME=proc
+      CUSTOM_PROCPS_NAME=procps
    fi
 
    # XXX: no pkg-config and no procps-config means we need to
diff -Naurp open-vm-tools.orig/configure.ac open-vm-tools/configure.ac
--- open-vm-tools.orig/configure.ac
+++ open-vm-tools/configure.ac
@@ -597,7 +597,7 @@ fi
 
 if test "$with_procps" = "yes"; then
    if test -z "$CUSTOM_PROCPS_NAME"; then
-      CUSTOM_PROCPS_NAME=proc
+      CUSTOM_PROCPS_NAME=procps
    fi
 
    # XXX: no pkg-config and no procps-config means we need to
